script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_1_1.txt

```diff
@@ -1,30 +1,30 @@
 <Page1>
 Unfortunately, I don't have anything to talk to her about.
-She doesn't seem to care that I'm at a loss for words and keeps smiling.
+She doesn't seem to mind that I'm at a loss for words and keeps smiling.
 <Page2>
 "Um... do you go to school by train, Senpai?"
 "Well, yes. I use it when I have to.
 What about you, Tohno-kun? Is your house far from here?"
 <Page3>
-"Yes, it's pretty far, near the station at the start of the line.
-But today is the last day for me to go to school by train."
+"Yes, it's pretty far, near the station at the start of this line.//line or route
```

Comment:
I'd revert. Pretty sure "start/end of the line" is how I'd refer to the extremes of a train line.